js/scripts.js

```diff
@@ -230,27 +251,50 @@ function placeMarkers(institutions) {
                 var offsetPerStudent = 40 * scale;
 
                 worldCoordinate.y -= defaultOffset +
-                    (offsetPerStudent * Math.min(5, this.students.length));
+                    (offsetPerStudent * Math.min(5, findSchoolFromLatLng(institutions, this.position).students.length));
                 worldCoordinate.y = Math.max(0, worldCoordinate.y);
 
                 var latLng = map.getProjection().fromPointToLatLng(worldCoordinate);
                 map.panTo(latLng);
             }
         });
-        marker.setMap(map);
         markers.push(marker);
     }
+
+    // Color markers according to the # of students attending (heatmap type coloring)
+    for(let marker of markers) {
+        institution = findSchoolFromLatLng(institutions, marker.position);
+        let numStudents = institution.students.length;
+        let color;
+        // Bucket 1: 1 student attending
+        if(numStudents <= 1) {
+            color = "#ff8282";
+        // Bucket 2: 2-5 students attending
+        } else if(numStudents <= 5) {
+            color = "#ff2d2d";
+        // Bucket 3: 6+ students attending
+        } else {
+            color = "#c80000";
+        }
+        const pinViewBackground = new google.maps.marker.PinView({
+            background: color,
+        });
```

Comment:
While I think the current coloring is good, on the darker red backgrounds the [glyph color](https://developers.google.com/maps/documentation/javascript/advanced-markers/basic-customization) begins to look just like the background color, making the marker look a bit different than the rest (when the background color is lighter)? Not sure if darkening the glyph proportionally would look good or not.

Reply:
Hi andrew this is sean and i love you very much